CMakeLists.txt

```diff
@@ -1483,7 +1483,33 @@ if(ANDROID)
       endif()
     endif()
   endif()
+elseif(OHOS)
+  # disable warnings from the toolchain
+  sdl_compile_options(PRIVATE "-Wno-unused-command-line-argument")
 
+  set(CMAKE_C_COMPILER_EXTERNAL_TOOLCHAIN "")
+  set(CMAKE_CXX_COMPILER_EXTERNAL_TOOLCHAIN "")
+
+  find_library(OHOS_ACE_LIBRARY NAMES libace_napi.z.so ace)
+  find_library(OHOS_LOG_LIBRARY NAMES libhilog_ndk.z.so hilog)
+  find_library(OHOS_NDK_LIBRARY NAMES libace_ndk.z.so ndk)
+  find_library(OHOS_RAWFILE_LIBRARY NAMES librawfile.z.so rawfile)
+  find_library(OHOS_PIXELMAP_LIBRARY NAMES libpixelmap_ndk.z.so pixelmap)
```

Comment:
The problem with linking to results of `find_library` is that the resulting SDL3 CMake package will only work on the build machine (for example I'm sure you installed the ohos sdk in a different place as ci does).
[`install(EXPORT)`](https://cmake.org/cmake/help/latest/command/install.html#export) will complain about this.

For the NGage toolchain toolchain, I [added interface libraries for certain system libraries](https://github.com/ngagesdk/ngage-toolchain/blob/main/cmake/ngage-toolchain-common.cmake#L76-L98).
But since openharmany is (probably) a little more maintained then n-gage, perhaps its sdk should make sure to provide a single library, or symbolic links?